recipes-openxt/openxt/openxt-measuredlaunch/seal-system

```diff
@@ -128,12 +128,12 @@ forward)
     # PCR8 holds the hashes of the other critical components of OpenXT's boot:
     #  openxt.cfg, the initrd and the XSM policy.
     # If PCR8 is all 0 we were not booting with UEFI
-    if [ "${tpm2}" -ne 0 ]; then
+    if [ "${tpm2}" -eq 0 ]; then
         pcr8=$(tpm2_listpcrs | grep PCR_08 | cut -f2 -d: | tr -d ' ' | tail -n1)
         pcr17=$(tpm2_listpcrs | grep PCR_17 | cut -f2 -d: | tr -d ' ' | tail -n1)
     else
-        pcr8=$(grep PCR-08 ${TPM_DEV}/device/pcrs | cut -f2 -d: | tr -d ' ')
-        pcr17=$(grep PCR-17 ${TPM_DEV}/device/pcrs | cut -f2 -d: | tr -d ' ' | tr '[:upper:]' '[:lower:]')
+        pcr8=$(grep PCR-08 ${TPM_DEV}/pcrs | cut -f2 -d: | tr -d ' ')
+        pcr17=$(grep PCR-17 ${TPM_DEV}/pcrs | cut -f2 -d: | tr -d ' ' | tr '[:upper:]' '[:lower:]')
```

Comment:
These two chunks should be PR'ed against stable-8 to fix the forward sealing. Would you mind sending a PR with that commit cherry-picked?